app/assets/stylesheets/hyrax/_dashboard.scss

```diff
@@ -123,7 +123,7 @@ ul.collapsing > li > a {
   padding-left: 40px;
 }
 
-.collapse-toggle {
+li > .collapse-toggle {
```

Comment:
Sounds like a better approach. I was trying too hard to walk lightly on the code. Code updated. I'll push the class to hyku as well if this PR gets merged.